po/de.po

```diff
@@ -598,31 +598,31 @@ msgstr "Kein CID mit dem Namen %s"
 
 #, c-format
 msgid "Unparseable include on line %d of %s"
-msgstr "Nicht parsebarer Include in Zeile %d in %s"
+msgstr "Unlesbares Einzufügendes in Zeile %d in %s"
 
 #, c-format
 msgid "Include filename too long on line %d of %s"
-msgstr "Include-Dateiname zu lang in Zeile %d in %s"
+msgstr "Name der einzufügenden Datei zu lang in Zeile %d in %s"
```

Comment:
This actually needs to be updated in crowdin, I'll find a way of getting it there

Reply:
I'll fix this once merged